### ISSUE TITLE
PoC: Cloudflare bulk redirects

### ISSUE THREAD

**Issue by robertu7** [thematters/developer-resource#365]
### Todos

- https://www.notion.so/matterslab/SEO-migration-a4a76247023e4a7ab713708fbd5eae31

### Downstream Tasks
* https://github.com/thematters/developer-resource/issues/346

### Story
- https://github.com/thematters/developer-resource/issues/372

**Comment by tx0c** [comment]
https://developers.cloudflare.com/workers/examples/bulk-redirects/

**Comment by tx0c** [comment]
all `500k` articles history has `27,052` rows of slug match beginning with numbers;

```console
➸ time npx wrangler deploy --minify
 ⛅️ wrangler 3.26.0
-------------------
Total Upload: 2997.86 KiB / gzip: 710.76 KiB
Uploaded bulk-redirects (2.30 sec)
Published bulk-redirects (1.65 sec)
  web-develop.matters.town/@Red_Lin/2023* (zone name: matters.news)
  matters.town/@Red_Lin/2023* (zone name: matters.town)
Current Deployment ID: 2ab4145e-56d5-4f65-9553-31c4b079a77b

real	0m5.917s
user	0m1.379s
sys	0m0.519s
```
